tests/transformers/test_transformers_model_export.py

```diff
@@ -2542,13 +2527,18 @@ def test_instructional_pipeline_with_prompt_in_output(model_path):
                 "inputs": '[{"name": "sequences", "type": "string"}, {"name": '
                 '"candidate_labels", "type": "string"}, {"name": '
                 '"hypothesis_template", "type": "string"}]',
-                "outputs": '[{"type": "string"}]',
+                "outputs": '[{"name": "sequence", "type": "string"}, {"name": "labels", '
+                '"type": "string"}, {"name": "score", "type": "double"}]',
```

Comment:
This is to match the output of `ModelSignature.to_dict()`. The alternative might be to construct the ModelSignature instance results direclty to a dict instead of a JSON encoded representation, but that will probably make this test far more complicated to follow.